.gitignore

```diff
@@ -33,3 +33,7 @@ tmp/
 
 # Logs
 *.log
+
+# Nearlib files & coverage
+nearlib/neardev/
```

Comment:
Put this into `nearlib/.gitignore` so that we can move `nearlib` and settings still apply?